aodntools/timeseries_products/aggregated_timeseries.py

```diff
@@ -349,6 +349,7 @@ def main_aggregator(files_to_agg, var_to_agg, site_code, input_dir='', output_di
                             (var_to_agg + "-" + product_type),
                             ('END-'+ time_end_filename), 'C-' + current_utc_timestamp(DATESTAMP_FORMAT)]) + '.nc'
     ncout_path = os.path.join(output_dir, output_name)
+    os.close(fd)
```

Comment:
:+1: That makes sense.
In fact, since we don't use this file descriptor to write to the file (instead we just use the `temp_outfile` as a name for a new netCDF4 file - [line 239](https://github.com/aodn/python-aodntools/blob/c3bad45d406bcc19af4ba67431ad8b378c8db114/aodntools/timeseries_products/aggregated_timeseries.py#L239)), you could even close it immediately after the call to `tempfile.mkstemp()`